packages/gafl-webapp-service/src/pages/order-complete/order-complete/route.js

```diff
@@ -58,7 +58,12 @@ const postalFulfilment = permission => {
   }
 }
 
-const isRecurringPayment = permission => process.env.SHOW_RECURRING_PAYMENTS?.toLowerCase() === 'true' && permission.isRecurringPayment
+const isRecurringPayment = transaction => {
+  if (process.env.SHOW_RECURRING_PAYMENTS?.toLowerCase() === 'true' && transaction.agreementId) {
+    return true
+  }
+  return false
+}
```

Comment:
I think this would be simpler if it were:
```
const isRecurringPayment = transaction => (process.env.SHOW_RECURRING_PAYMENTS?.toLowerCase() === 'true' && !!transaction.agreementId)
```

Reply:
Good shout, fixed!